#! /bin/sh -e
##
## All lines beginning with `## DP:' are a description of the patch.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad asterisk-1.2.0.dfsg~/channels/chan_sip.c asterisk-1.2.0.dfsg/channels/chan_sip.c
--- asterisk-1.2.0.dfsg~/channels/chan_sip.c	2005-12-05 23:57:46.000000000 +0000
+++ asterisk-1.2.0.dfsg/channels/chan_sip.c	2005-12-05 23:59:49.000000000 +0000
@@ -50,7 +50,7 @@
 
 #include "asterisk.h"
 
-ASTERISK_FILE_VERSION(__FILE__, "$Revision: 1.913 $")
+ASTERISK_FILE_VERSION(__FILE__, "$Revision: 1.914 $")
 
 #include "asterisk/lock.h"
 #include "asterisk/channel.h"
@@ -4267,7 +4267,7 @@
 			 sample_rate);
 	if (codec == AST_FORMAT_G729A)
 		/* Indicate that we don't support VAD (G.729 annex B) */
-		ast_build_string(a_buf, a_size, "a=fmtp:%d annexb=no", rtp_code);
+		ast_build_string(a_buf, a_size, "a=fmtp:%d annexb=no\r\n", rtp_code);
 }
 
 static void add_noncodec_to_sdp(const struct sip_pvt *p, int format, int sample_rate,
